packages/types/CHANGELOG.md

```diff
@@ -6,6 +6,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [3.12.0] - 2024-11-21
+### Added
+- Update polkadot/api library to 14 version
```

Comment:
This should be in the unreleased section

Reply:
I see, fixed